modules/python/python-example.conf

```diff
@@ -1,4 +1,4 @@
-@version: 3.28
+@version: current
```

Comment:
again, we shouldn't use `current` in configs we are using as examples. internal test usage is ok.